inference-engine/ie_bridges/java/cpp/blob.cpp

```diff
@@ -0,0 +1,126 @@
+#include <inference_engine.hpp>
+#include "openvino_java.hpp"
+#include "jni_common.hpp"
+
+using namespace InferenceEngine;
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_Blob_GetTensorDesc(JNIEnv *env, jobject obj, jlong addr)
+{
+    static const char method_name[] = "GetTensorDesc";
+    try
+    {
+        Blob::Ptr *output = reinterpret_cast<Blob::Ptr *>(addr);
+        TensorDesc *tDesc = new TensorDesc((*output)->getTensorDesc());
+
+        return (jlong)tDesc;
+    }
+    catch (const std::exception &e)
+    {
+        throwJavaException(env, &e, method_name);
+    }
+    catch (...)
+    {
+        throwJavaException(env, 0, method_name);
+    }
+
+    return 0;
+}
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_Blob_GetBlob(JNIEnv *env, jobject obj, jlong tensorDescAddr)
+{
+    static const char method_name[] = "GetBlob";
+    try
+    {
+        TensorDesc *tDesc = (TensorDesc *)tensorDescAddr;
+
+        Blob::Ptr *blob = new Blob::Ptr();
+        *blob = make_shared_blob<uint8_t>(*tDesc);
+
+        return (jlong)blob;
+    }
+    catch (const std::exception &e)
+    {
+        throwJavaException(env, &e, method_name);
+    }
+    catch (...)
+    {
+        throwJavaException(env, 0, method_name);
+    }
+
+    return 0;
+}
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_Blob_BlobByte(JNIEnv *env, jobject obj, jlong tensorDescAddr, jbyteArray data)
+{
+    static const char method_name[] = "BlobByte";
+    try
+    {
+        TensorDesc *tDesc = (TensorDesc *)tensorDescAddr;
+
+        Blob::Ptr *blob = new Blob::Ptr();
+        *blob = make_shared_blob<uint8_t>((*tDesc), (uint8_t *)(env->GetByteArrayElements(data, 0)));
```

Comment:
This is invalid usage. `GetByteArrayElements` allocates a memory which should be released by `ReleaseByteArrayElements`. So for Blob creation I'd like to recommend to do this:
```java
        *blob = make_shared_blob<uint8_t>((*tDesc));
        (*blob)->allocate();
        env->GetByteArrayRegion(data, 0, (*blob)->byteSize(), (*blob)->buffer());
```